extensionsGallery-insider.json

```diff
@@ -1272,14 +1272,14 @@
 					},
 					"versions": [
 						{
-							"version": "0.8.0",
+							"version": "1.0.0",
 							"lastUpdated": "10/10/2019",
```

Comment:
the "lastUpdated" date should be updated too

Reply:
Thanks for catching! done.